tensorboard/webapp/widgets/histogram/histogram_component.ts

```diff
@@ -12,6 +12,8 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
+import '../../tb_polymer_interop_types';
```

Comment:
Are we now able to drop the `as VzHistogramTimeSeries;` on L79 in this file?

Reply:
Ah drats. Forgot to add those changes to the commit. Thanks.